tests/unit/test_finops.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import pytest
+
+from cost_reports import finops
```

Comment:
I would expect to see a test that takes in a sample cost-and-usage report, produces a summary report, and checks that it's valid.

Reply:
I've expanded the tests